docs/physics/setup/model.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
<p>Typo/confusing sentence at "Using geometry..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Oops, I didn't finish that sentence